### PR TITLE
Sanitise text - nbsp and br

### PR DESCRIPTION
**Pull request by bcazur**
Fix for #12

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -421,4 +422,12 @@ export class HTMLText extends Sprite
         this._resolution = value;
         this.dirty = true;
     }
+
+    sanitiseText(text){
+        // Sanitise text - replace <br> with <br/>, &nbsp; with &#160;
+        // See discussion here: https://www.sitepoint.com/community/t/xhtml-1-0-transitional-xml-parsing-error-entity-nbsp-not-defined/3392/3
+        text = text.replace(new RegExp('<br>'.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&'), 'g'), '<br/>');
```
Comment (bigtimebuddy) [inline src/index.js]: This regex seems a little verbose for what this is. Could you do this simpler instead:

```
/<br>/gi
```

We also may want to handle other non terminating single tags like `<hr>` as well as attributes like style. But for now this seems simple enough.

Comment (bcazur) [reply src/index.js]: I agree. Simplified regex and added <hr>. We can add more if we have some samples, these are the only ones I've hit until now.
Thanks!

**Comment by bigtimebuddy** [comment]
Can you add the `i` flag to the regexp to handle `<BR>` too?

**Comment by bcazur** [comment]
> Can you add the `i` flag to the regexp to handle `<BR>` too?

Done now.

**Comment by bigtimebuddy** [comment]
Thank you!

**Pull request merged**